### PR TITLE
Function Entry Package

### PR DESCRIPTION
**Pull request by elodszopos**
Also a hardcoded path here in package json that will need to be fixed. Please look it over @edeustace and let me know! Thanks!

### CODE REVIEW

File: packages/function-entry/configure/src/feedback-mapper.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
+export const modelToFeedbackConfig = model => {
```
Comment (edeustace) [inline packages/function-entry/configure/src/feedback-mapper.js]: we have something similar in `text-entry` - can we re-use that? could import like so: 
`import { feedbackMapper } from '@pie-lib/config-ui;`?

File: packages/function-entry/configure/src/model-config.jsx [hunk 2]
```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Typography } from 'material-ui';
+import { InputCheckbox } from '@pie-lib/config-ui';
+import { withStyles } from 'material-ui/styles';
+import Box from './box';
+
+class RawModelConfig extends React.Component {
+  static propTypes = {
+    classes: PropTypes.object,
```
Comment (edeustace) [inline packages/function-entry/configure/src/model-config.jsx]: all `.isRequired`?

File: packages/function-entry/controller/src/index.js [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+const log = debug('@pie-element:function-entry:controller');
+
+const process = (v, ignoreWhitespace) => {
+    let out = v ? v.trim() : '';
+    out = !ignoreWhitespace ? out : out.replace(/ /g, '');
+    return out;
+};
+
+const isResponseCorrect = (correctResponse, model, value) => {
+    const processedValue = process(value, model.ignoreSpacing);
```
Comment (edeustace) [inline packages/function-entry/controller/src/index.js]: We'll need to do more than just a string compare here, we were talking about using `mathjs.simplify`?

File: packages/function-entry/controller/src/index.js [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+
+    return processedValue  === correctResponse;
+};
+
+export function model(question, session, env) {
```
Comment (edeustace) [inline packages/function-entry/controller/src/index.js]: Can we use 2 spaces for indents?

File: packages/function-entry/configure/src/feedback-mapper.js (hunk 1 above)
Comment (elodszopos) [reply packages/function-entry/configure/src/feedback-mapper.js]: At this point, the divergence between the two is quite substantial. There are different property names, different properties altogether and a slightly different structure. If perhaps a third component might need something similar further down the road we could maybe come up with a helper function that would help generate the structures that we need.

Comment (edeustace) [reply packages/function-entry/configure/src/feedback-mapper.js]: 👍

**Pull request merged**